### PR TITLE
Notifications for StorIOContentProvider

### LINKED ISSUE #116: StorIOContentResolver should handle notifications
And provide same API as `StorIOSQLiteDb`

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #116 

@nikitin-da PTAL

I am thinking how to write tests for that, it'll be separate PR

### CODE REVIEW

**Comment by artem-zinnatullin** [comment]
Looks like emulator freezed, I created new one

**Comment by artem-zinnatullin** [comment]
retest this please

File: storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/impl/DefaultStorIOContentResolver.java [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+         */
+        @NonNull
+        public CompleteBuilder contentResolver(@NonNull ContentResolver contentResolver) {
+            this.contentResolver = contentResolver;
+            return new CompleteBuilder(this);
+        }
+    }
+
+    /**
+     * Compile-time safe part of builder for {@link DefaultStorIOContentResolver}
```
Comment (nikitin-da) [inline storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/impl/DefaultStorIOContentResolver.java]: Hm, nice comment

**Comment by nikitin-da** [comment]
Awesome!

**Pull request merged**